### PR TITLE
Added myself to the list.

### PR DESCRIPTION
**Pull request by Khukhunashvili**

### CODE REVIEW

File: _winners/saba-khukhunashvili.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Saba Khukhunashvili
+year: 2018
+org: JBoss Community
+country: GEO
+links:
+    -
+        icon: global
```
Comment (Albert221) [inline _winners/saba-khukhunashvili.md]: `globe`, not `global`

File: _winners/saba-khukhunashvili.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+name: Saba Khukhunashvili
+year: 2018
+org: JBoss Community
+country: GEO
```
Comment (Albert221) [inline _winners/saba-khukhunashvili.md]: Use Emoji flag here:  🇬🇪

**Review by Albert221 — CHANGES_REQUESTED** [review]

**Comment by Khukhunashvili** [comment]
Done.

File: _winners/saba-khukhunashvili.md [hunk 3]
```diff
@@ -2,10 +2,10 @@
 name: Saba Khukhunashvili
 year: 2018
 org: JBoss Community
-country: GEO
+country: georgia
```
Comment (Albert221) [inline _winners/saba-khukhunashvili.md]: Please, use Georgia flag emoji: 🇬🇪

**Review by Albert221 — CHANGES_REQUESTED** [review]

File: _winners/saba-khukhunashvili.md (hunk 3 above)
Comment (Khukhunashvili) [reply _winners/saba-khukhunashvili.md]: It's :georgia :, isn't it? :georgia:
Should I capitalize the first letter?

Comment (Khukhunashvili) [reply _winners/saba-khukhunashvili.md]: o shout, i forgot : 🤣

Comment (Albert221) [reply _winners/saba-khukhunashvili.md]: Browsers don't change those phrases into emojis, just paste there https://emojipedia.org/flag-for-georgia/

Comment (Khukhunashvili) [reply _winners/saba-khukhunashvili.md]: Sorry for misunderstanding. It should work now.

**Review by Albert221 — APPROVED** [review]

**Pull request merged**